### PR TITLE
[Configuration] help text for subpages Project and Subproject

### PR DESCRIPTION
**Pull request by knoll-alex**
created help text files for the subpages (both in this one PR)

#### Testing instructions

1. review copy
2. see screenshot for rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576
* https://github.com/aces/Loris/issues/5732
* https://github.com/aces/Loris/issues/5728

Screenshot 2019-11-24 19 32 40
Screenshot 2019-11-24 19 32 18

### CODE REVIEW

File: modules/configuration/help/project.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Configuration: Project
+
+Use this page to manage the configuration of existing projects, or to add a new one. Your projects are listed on the left side of the screen. Click on any project name to edits its settings, and click **Save**. 
+
+Click **New ProjectID** from the left side to crate a new project. Populate the fields and click **Save**. Your new project should now appear in the list, after you refresh the page. 
+
+Note: Clicking save more than once will register a duplicate ProjectID. If you create an extra ID, you have to delete it from the database manually with an sql command.\n\nDefine all projectID-subprojectID relationships by populating the project_rel table.
```
Comment (johnsaigle) [inline modules/configuration/help/project.md]:
```suggestion
Note: Clicking save more than once will register a duplicate ProjectID. If you create an extra ID, you have to delete it from the database manually with an sql command.

Define all projectID-subprojectID relationships by populating the project_rel table.
```

File: modules/configuration/help/subproject.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Configuration: Subproject
+
+Use this page to manage the configuration of existing subprojects, or to add a new one. Your subprojects are listed on the left side of the scren. Click on any subproject name to edit its settings, and click **Save.**
+
+Click **New SubprojectID** to create a new subproject. Populate the fields and click **Save**. Your new subproject should now appear in the list after you refresh the page.
+
+Note: Clicking save more than once will register a duplicate SubprojectID.  If you create an extra ID, you have to delete it from the database manually with an sql command.\n\nDefine all projectID-subprojectID relationships by populating the project_rel table.
```
Comment (johnsaigle) [inline modules/configuration/help/subproject.md]:
```suggestion
Note: Clicking save more than once will register a duplicate SubprojectID.  If you create an extra ID, you have to delete it from the database manually with an sql command.

Define all projectID-subprojectID relationships by populating the project_rel table.
```

File: modules/configuration/help/subproject.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Configuration: Subproject
+
+Use this page to manage the configuration of existing subprojects, or to add a new one. Your subprojects are listed on the left side of the scren. Click on any subproject name to edit its settings, and click **Save.**
+
+Click **New SubprojectID** to create a new subproject. Populate the fields and click **Save**. Your new subproject should now appear in the list after you refresh the page.
+
+Note: Clicking save more than once will register a duplicate SubprojectID.  If you create an extra ID, you have to delete it from the database manually with an sql command.
```
Comment (johnsaigle) [inline modules/configuration/help/subproject.md]: @ridz1208 Do you know whether this is true? If so it sounds like a pretty basic SQL bug that we should fix.

Comment (ridz1208) [reply modules/configuration/help/subproject.md]: no longer the case, this is an old bug

File: modules/configuration/help/project.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Configuration: Project
+
+Use this page to manage the configuration of existing projects, or to add a new one. Your projects are listed on the left side of the screen. Click on any project name to edits its settings, and click **Save**. 
+
+Click **New ProjectID** from the left side to crate a new project. Populate the fields and click **Save**. Your new project should now appear in the list, after you refresh the page. 
+
+Note: Clicking save more than once will register a duplicate ProjectID. If you create an extra ID, you have to delete it from the database manually with an sql command.
```
Comment (johnsaigle) [inline modules/configuration/help/project.md]:
```suggestion
```

File: modules/configuration/help/subproject.md (hunk 3 above)
Comment (johnsaigle) [inline modules/configuration/help/subproject.md]:
```suggestion
```

**Comment by christinerogers** [comment]
@knoll-alex  2 changes requested in this pr, by me and John (scroll above ^)

File: modules/configuration/help/project.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Project
+
+Use this page to manage the configuration of existing projects, or to add a new one. Your projects are listed on the left side of the screen. Click on any project name to edits its settings, and click **Save**. 
+
+Click **New ProjectID** from the left side to crate a new project. Populate the fields and click **Save**. Your new project should now appear in the list, after you refresh the page. 
+
+
+Define all projectID-subprojectID relationships by populating the project_rel table.
```
Comment (driusan) [inline modules/configuration/help/project.md]: I think this sentence in the help text is going to confuse users.. how do they populate the table? Do they need a dev to do it?

File: modules/configuration/help/subproject.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Subproject
+
+Use this page to manage the configuration of existing subprojects, or to add a new one. Your subprojects are listed on the left side of the scren. Click on any subproject name to edit its settings, and click **Save.**
+
+Click **New SubprojectID** to create a new subproject. Populate the fields and click **Save**. Your new subproject should now appear in the list after you refresh the page.
+
+
+Define all projectID-subprojectID relationships by populating the project_rel table.
```
Comment (driusan) [inline modules/configuration/help/subproject.md]: same here

File: modules/configuration/help/project.md (hunk 5 above)
Comment (johnsaigle) [reply modules/configuration/help/project.md]: I agree, these lines can probably be deleted. It's more appropriate to the module README or a wiki page.

Comment (knoll-alex) [reply modules/configuration/help/project.md]: deleted

File: modules/configuration/help/subproject.md (hunk 6 above)
Comment (knoll-alex) [reply modules/configuration/help/subproject.md]: deleted

**Comment by knoll-alex** [comment]
deleted those lines - this should be good to go now

**Pull request merged**